script/check-posix-build-cmake

```diff
@@ -48,6 +48,11 @@ build()
 
     options+=(
         "-DOT_TREL=ON"
+        "-DOT_BORDER_ROUTER=ON"
+        "-DOT_BORDER_ROUTING=ON"
+        "-DOT_SRP_SERVER=ON"
+        "-DOT_SERVICE=ON"
+        "-DOT_ECDSA=ON"
```

Comment:
`OT_BORDER_ROUTER`, `OT_SERVICE` and `OT_ECDSA` are already included in `OT_POSIX_SIM_COMMON_OPTIONS`(https://github.com/openthread/openthread/blob/main/script/cmake-build#L73) and should be set to `ON` when calling `cmake-build posix`.